test/e2e/helloworld_shell_test.go

```diff
@@ -97,15 +94,15 @@ func TestHelloWorldFromShell(t *testing.T) {
 		t.Fatalf("Failed to close new manifest file: %v", err)
 	}
 
-	logger.Infof("Manifest file is %q", newYamlFilename)
-	logger.Info("Deploying using kubectl")
+	t.Logf("Manifest file is %q", newYamlFilename)
```

Comment:
While you're at it, may be join into one statement?
`Deploying using kubectl and using manifest file: %s`?